src/GuppyDataExplorer/index.jsx

```diff
@@ -48,16 +48,25 @@ const defaultFileConfig = {
   dropdowns: {},
 };
 
-const guppyExplorerConfig = [
+let guppyExplorerConfigs = [
   _.merge(defaultConfig, config.dataExplorerConfig),
   _.merge(defaultFileConfig, config.fileExplorerConfig),
 ];
-
-const routes = [
+let routes = [
   '/explorer',
   '/files',
 ];
 
+const converNameToLink = (name) => {
+  const linkName = name.replace(' ', '_').toLowerCase();
+  return `/explorer/${linkName}`;
+};
+if (config.explorerConfigs) { // using new multi-tab configuration
```

Comment:
use `Object.keys(config.explorerConfigs).length > 1` to check?

otherwise it will error and not fall back to use old configs if it has `explorerConfigs : {}`